webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/GroupService.scala

```diff
@@ -39,16 +36,6 @@ final case class GroupService(
       status = knoraGroup.status.value,
       selfjoin = knoraGroup.hasSelfJoinEnabled.value,
     )
-
-  def toKnoraGroup(group: Group): KnoraGroup =
-    KnoraGroup(
-      id = GroupIri.unsafeFrom(group.id),
-      groupName = GroupName.unsafeFrom(group.name),
-      groupDescriptions = GroupDescriptions.unsafeFrom(group.descriptions),
-      status = GroupStatus.from(group.status),
-      belongsToProject = group.project.map(it => ProjectIri.unsafeFrom(it.id)),
-      hasSelfJoinEnabled = GroupSelfJoin.from(group.selfjoin),
-    )
```

Comment:
Could you bring it back please? It is unused not, but will be soon - group slice WIP.

Reply:
Please readd it yourself when in use and ideally tested.